docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,66 @@
+
+Some modules have more than one Help text file to correspond with subpages. These files exist in the same help/ folder, but are named differently - they must match exactly the php/file name that spawns the page. For example, the Imaging Browser module has a subpage called “ViewSession”. Its help file is stored as: `modules/imaging_browser/help/viewsession.md`. 
+
+## Markdown
+
+Formatting should follow markdown guidelines, with some exceptions. This is because LORIS’s current markdown template may not cover all possible markdown features (i.e. bullets). Help text should not include HTML tags for security reasons. When creating a markdown file, use only the following styles, or else talk to the LORIS technical team about expanding our Markdown features:
+
+* `*Italics*`
+* `**Bold**`
+* `***Italics and Bold***`
+* `# Heading 1` up to `###### Heading 6`
+  * Use `#H1` for the name of the module at the beginning of the help text
+  * Use `#H2` for any subheadings
+  * Use `#H4` for sub-subheadings
+* Avoid using HTML line break (`<br>`) tags—simply use physical line breaks (spaces between lines) where necessary. 
+  * Using `<br>` might be necessary only in lists of items
+  
+  ## When to Use Emphasis
+  
+Whether a user knows it or not, they will come to recognize styles that are consistent throughout the text. This way, their eyes can jump to a part of the text that addresses their issue, if they don’t need to read the whole thing. Follow these rules on using emphasis:
+
+* *Italicize* when referring to a specific place on the screen
+  * i.e. in the Selection Filters section
+  * i.e. in the Metadata column of the table
+  * i.e. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * i.e. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * i.e. In the **Imaging Browser** module
+    * i.e. Click the **Upload** tab
+  * You can also bold when defining features in a list, to separate the term being defined from the definition itself, i.e. when describing columns in a table:<br>
+     **Column heading 1**: description<br>
+     **Column heading 2**: description<br>
+     Etc.
+     
+## Tone
+
+Yes, LORIS is used for high-level research and yes, the people using LORIS are very smart and yes, we want people in the scientific community to take LORIS seriously. At the same time, the easier any user finds a text to read, the more they’ll retain and use the information. And a friendly tone will help them understand that LORIS is built and maintained by humans. There’s a lot of research that shows people will engage and have more patience with a piece of software when they’re reminded that there’s more than just a computer behind it. Using a more conversational tone will subtly have this effect. We’re not talking about humour and exclamation marks—just small tactics that make content as easy to understand as possible.
```

Comment:
Do you have any references for "a lot of research" (both out of curiousity, and in case someone asks in the future) that could be referenced here?

Reply:
Hi @driusan - I removed this line (along with the rest of that paragraph), but still happy to provide some context to this claim. I added an article at the end that talks about Plain Language, and cites a usability study that was conducted in scientific fields. I hope this will suffice!